src/map/navigation_container.h

```diff
@@ -0,0 +1,199 @@
+﻿//
+// Copyright (c) 2009-2010 Mikko Mononen memon@inside.org
+//
+// This software is provided 'as-is', without any express or implied
+// warranty.  In no event will the authors be held liable for any damages
+// arising from the use of this software.
+// Permission is granted to anyone to use this software for any purpose,
+// including commercial applications, and to alter it and redistribute it
+// freely, subject to the following restrictions:
+// 1. The origin of this software must not be misrepresented; you must not
+//    claim that you wrote the original software. If you use this software
+//    in a product, an acknowledgment in the product documentation would be
+//    appreciated but is not required.
+// 2. Altered source versions must be plainly marked as such, and must not be
+//    misrepresented as being the original software.
+// 3. This notice may not be removed or altered from any source distribution.
+//
+
+// Modified: Copyright (c) 2023 AirSkyBoat Dev Team
```

Comment:
Would have liked you to try and do a git-rename of the previous `navmesh.h/.cpp` files so that all of the DSP/TPZ/LSB history of the file doesn't get erased. How much of that original code is still in here? I only really check to see if my `sameFloor` code was here or not. 

There's a lot of code we added to improve how the navmesh works, I hope its still in here. (code: https://github.com/project-topaz/topaz/pull/1458)

Reply:
Rebased the branch into a single commit, used git mv to fix this.